one_fm/setup.py

```diff
@@ -97,6 +109,11 @@ def remove_property_setter(property_setters):
 
 def delete_workflows():
 	delete_workflow(get_workflow_json_file("erf.json"))
+	delete_workflow(get_workflow_json_file("shift_request.json"))
 
 def delete_assignment_rules():
 	delete_assignment_rule(get_assignment_rule_json_file("roster_post_action_site_supervisor.json"))
+	delete_assignment_rule(get_assignment_rule_json_file("shift_request_pending_approval_project_manager.json"))
+	delete_assignment_rule(get_assignment_rule_json_file("shift_request_pending_approval_reports_to.json"))
```

Comment:
Use this approach https://docs.google.com/document/d/1Yef_G0x8YSQOwv2y2Fr0yfnpOqBjeIOFBu5jIM_ZFrw/edit?tab=t.0#heading=h.rc1o5gbb773u